src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,76 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  position: relative;
+  background-color: var(--sbb-color-white-default);
+  width: 100%;
+  border-radius: var(--sbb-border-radius-4x);
+  padding-inline: var(--sbb-spacing-responsive-xxs-small);
+  overflow: hidden;
+  transition-property: box-shadow var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
+
+  @include mq($from: medium) {
+    padding-inline: var(--sbb-spacing-responsive-xxs-medium);
+  }
+}
+
+:host([size='xs']) {
+  padding-block: var(--sbb-spacing-fixed-3x);
+}
+
+:host([size='s']) {
+  padding-block: var(--sbb-spacing-responsive-xxxs-small);
+
+  @include mq($from: medium) {
+    padding-block: var(--sbb-spacing-responsive-xxxs-medium);
+  }
+}
+
+:host([size='m']) {
+  padding-block: var(--sbb-spacing-fixed-3x);
+}
+
+:host([size='l']) {
+  padding-block: var(--sbb-spacing-fixed-4x);
+}
+
+:host([size='xl']) {
+  padding-block: var(--sbb-spacing-responsive-xxs-small);
+
+  @include mq($from: medium) {
+    padding-block: var(--sbb-spacing-responsive-xxs-medium);
+  }
+}
+
+:host(.card__badge[size='m']),
+:host(.card__badge[size='l']),
+:host(.card__badge[size='xl']) {
+  padding-block-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host([size='xxl']) {
+  padding-block: var(--sbb-spacing-responsive-s-small);
+
+  @include mq($from: medium) {
+    padding-block: var(--sbb-spacing-responsive-s-medium);
+  }
+}
+
+.card__content {
+  display: block;
+  transition: transform var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
+}
+
+:host(:hover) {
+  cursor: pointer;
+
+  .card__content {
+    transform: translateY(-2px);
```

Comment:
Should we maybe create a design token for the hover displacement?
What do you think? (@jeripeierSBB FYI)

Reply:
to compare, on button it's only 1 px shift. I suggest to use a var inside this component: 
--translate-y-content-hover: #{px-to-rem-build(-2)};

Reply:
Maybe could I change to something like [this](https://github.com/lyne-design-system/lyne-components/blob/7bb07725854fdfc3d52b97c2a3bc12b7b564a15f/src/components/sbb-tab-title/sbb-tab-title.scss#L58)? so:
`transform: translateY(calc(var(--sbb-border-width-2x) * -1));`

Reply:
for me it's okay to not use border width here